doc/api/buffer.md

```diff
@@ -103,8 +103,14 @@ string into a `Buffer` as decoding.
 
 * `'base64'`: [Base64][] encoding. When creating a `Buffer` from a string,
   this encoding will also correctly accept "URL and Filename Safe Alphabet" as
-  specified in [RFC 4648, Section 5][]. Whitespace characters such as spaces,
-  tabs, and new lines contained within the base64-encoded string are ignored.
+  specified in [RFC 4648, Section 5][]. Whitespace characters such as
+  spaces, tabs, and new lines contained within the base64-encoded string are
+  ignored.
```

Comment:
Nit: Unrelated change? It just alters the point at which the line is wrapped.

Reply:
✅